service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt

```diff
@@ -1944,6 +1944,28 @@ sealed interface Action {
                 .inUnit(),
             IsMobile(requirePinLogin = false).inUnit()
         ),
+        READ_CHILD_RESERVATIONS(
+            HasGlobalRole(ADMIN, SERVICE_WORKER),
+            HasUnitRole(
+                    UNIT_SUPERVISOR,
+                    STAFF,
+                    SPECIAL_EDUCATION_TEACHER,
+                    EARLY_CHILDHOOD_EDUCATION_SECRETARY
+                )
+                .inUnit(),
+            IsMobile(requirePinLogin = false).inUnit()
+        ),
+        READ_UNIT_RESERVATION_STATISTICS(
+            HasGlobalRole(ADMIN, SERVICE_WORKER),
+            HasUnitRole(
+                    UNIT_SUPERVISOR,
+                    STAFF,
+                    SPECIAL_EDUCATION_TEACHER,
+                    EARLY_CHILDHOOD_EDUCATION_SECRETARY
+                )
+                .inUnit(),
+            IsMobile(requirePinLogin = false).inUnit()
+        ),
```

Comment:
Is it necessary to add these permissions employee to roles if the enptoints are only used in mobile?

Reply:
Desktop employee permissions removed.